curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6443d0428d7c2402e9d78709.md

```diff
@@ -0,0 +1,209 @@
+---
+id: 6443d0428d7c2402e9d78709
+title: Step 51
+challengeType: 0
+dashedName: step-51
+---
+
+# --description--
+
+While staying inside the function named `showTime` you will create a `if` statement where it will check whether the constant `minutesAgo` is less than 60. If it is, then it will `return` the template literal saying `${minutesAgo}m ago`. What it does is that it will return the number of minutes ago the post was created.
```

Comment:
```suggestion
Within the function `showTime`, create an `if` statement that checks whether the constant `minutesAgo` is less than 60. If it is, then `return` the template literal `${minutesAgo}m ago`. This would return the number of minutes ago the post was created.
```

Reply:
added